From ed291458f4909bdb8ff7db564e33ad78dff1e7d7 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Mon, 30 Oct 2023 14:26:35 +0100
Subject: [PATCH 0022/1695] MAINTAINERS: update STMicroelectronics platforms
 and drivers entries

Add an entry for STM32MP2x platforms and STMicroelectronics drivers
with Etienne Carriere and myself as maintainers.

Update STM32MP1x platforms entry as ST drivers' maintenance is now
handle with the STMicroelectronics drivers entry.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git c8b9e2acf4475716d64b429653bc44facfa85053 MAINTAINERS: update STMicroelectronics platforms and drivers entries]
Change-Id: Ib6ae9ef4714eb717786932831a7a70f053cbd561
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
[etienne: cherry-picked from 2b7a5a6371cbba3035502bc16171b25691207a22]
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
---
 MAINTAINERS | 13 ++++++++++++-
 1 file changed, 12 insertions(+), 1 deletion(-)

diff --git a/MAINTAINERS b/MAINTAINERS
index b4805295e..c991f8ffc 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -267,14 +267,25 @@ R:	Etienne Carriere <etienne.carriere@foss.st.com> [@etienne-lms]
 S:	Maintained
 F:	core/arch/arm/plat-stm/
 
+STMicroelectronics drivers
+R:	Etienne Carriere <etienne.carriere@foss.st.com> [@etienne-lms]
+R:	Gatien Chevallier <gatien.chevallier@foss.st.com> [@GseoC]
+S:	Maintained
+F:	core/drivers/stm32_*
+
 STMicroelectronics stm32mp1
 R:	Etienne Carriere <etienne.carriere@foss.st.com> [@etienne-lms]
 S:	Maintained
 F:	core/arch/arm/plat-stm32mp1/
-F:	core/drivers/stm32_*
 F:	core/drivers/stm32mp15_huk.c
 F:	core/drivers/stpmic1.c
 
+STMicroelectronics stm32mp2
+R:	Etienne Carriere <etienne.carriere@foss.st.com> [@etienne-lms]
+R:	Gatien Chevallier <gatien.chevallier@foss.st.com> [@GseoC]
+S:	Maintained
+F:	core/arch/arm/plat-stm32mp2/*
+
 Texas Instruments AM43xx, AM57xx, DRA7xx, AM65x, J721E, J784S4, AM64x, AM62x
 R:	Andrew Davis <afd@ti.com> [@glneo]
 S:	Maintained
-- 
2.47.2

